packages/lesswrong/components/ea-forum/EAEmojisHeader.tsx

```diff
@@ -0,0 +1,584 @@
+import React, { FC, MouseEvent, ReactNode, createContext, useCallback, useContext, useEffect, useRef, useState } from "react";
```

Comment:
I really dislike this style of architecture when we just define everything in a single big file, but this is quick and hacky so whatever. At least it'll make it easier to remove next week.